Lumino/js/ControllerSettings.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { StyleSheet, View } from 'react-native';
+import ControllerTypeSelector from './ControllerTypeSelector';
+import ControllerSettingsForm from './ControllerSettingsForm.js';
```

Comment:
Wouldn't be better `import ControllerSettingsForm from './ControllerSettingsForm';`?